disco_aws_automation/resource_helper.py

```diff
@@ -37,6 +37,15 @@ def tag2dict(tags):
     return {tag.get('Key'): tag.get('Value') for tag in tags or {}}
 
 
+def dict2tag(dicts):
```

Comment:
This method takes a list of strings instead of a dict so maybe it should be renamed